python/samples/getting_started_with_agents/copilot_studio/step1_copilot_studio_agent_simple.py

```diff
@@ -0,0 +1,54 @@
+# Copyright (c) Microsoft. All rights reserved.
+
+import asyncio
+
+from microsoft.agents.copilotstudio.client import (
+    CopilotClient,
+)
+
+from semantic_kernel.agents import CopilotStudioAgent
+
+"""
+This sample demonstrates how to use the Copilot Studio agent to answer questions about physics.
+It does not use a thread to maintain context between user inputs.
+"""
+
+
+async def main() -> None:
+    client: CopilotClient = CopilotStudioAgent.setup_resources()
```

Comment:
maybe:
```suggestion
    client: CopilotClient = CopilotStudioAgent.create_client()
```
to keep it consistent with AzureAIAgent